src/features/defi/providers/foxy/components/FoxyManager/Withdraw/FoxyWithdraw.tsx

```diff
@@ -0,0 +1,585 @@
+                  {translate('modals.withdraw.withdrawFee')}
+                </AlertDescription>
+              </Alert>
+            }
+            onCancel={() => history.push('/')}
+            onConfirm={handleApprove}
+          />
+        )
+      case WithdrawPath.Confirm:
+        return (
+          <Confirm
+            onCancel={handleCancel}
+            headerText='modals.confirm.withdraw.header'
+            onConfirm={handleConfirm}
+            assets={[
+              {
+                ...asset,
+                color: '#FFFFFF',
+                cryptoAmount: state.withdraw.cryptoAmount,
+                fiatAmount: state.withdraw.fiatAmount
+              },
+              {
+                ...underlyingAsset,
+                color: '#FF0000',
+                cryptoAmount: state.withdraw.cryptoAmount,
+                fiatAmount: state.withdraw.fiatAmount
+              }
+            ]}
+          >
+            <Stack spacing={6}>
+              <Row>
+                <Row.Label>
+                  <Text translation='modals.confirm.withdrawFrom' />
+                </Row.Label>
+                <Row.Value fontWeight='bold'>
+                  <Text translation='defi.yearn' />
+                </Row.Value>
+              </Row>
+              <Row>
+                <Row.Label>
+                  <Text translation='modals.confirm.withdrawTo' />
+                </Row.Label>
+                <Row.Value>
+                  <MiddleEllipsis address={state.userAddress || ''} />
+                </Row.Value>
+              </Row>
+              <Row>
+                <Row.Label>
+                  <Text translation='modals.confirm.estimatedGas' />
+                </Row.Label>
+                <Row.Value>
+                  <Box textAlign='right'>
+                    <Amount.Fiat
+                      fontWeight='bold'
+                      value={bnOrZero(state.withdraw.estimatedGasCrypto)
+                        .div(`1e+${feeAsset.precision}`)
+                        .times(feeMarketData.price)
+                        .toFixed(2)}
+                    />
+                    <Amount.Crypto
+                      color='gray.500'
+                      value={bnOrZero(state.withdraw.estimatedGasCrypto)
+                        .div(`1e+${feeAsset.precision}`)
+                        .toFixed(5)}
+                      symbol={feeAsset.symbol}
+                    />
+                  </Box>
+                </Row.Value>
+              </Row>
+            </Stack>
+          </Confirm>
+        )
+      case WithdrawPath.Status:
+        return (
+          <TxStatus
+            onClose={handleCancel}
+            onContinue={handleViewPosition}
+            loading={state.loading}
+            continueText='modals.status.position'
+            closeText='modals.status.close'
+            statusText={statusText}
+            statusIcon={statusIcon}
```

Comment:
Here is where you'll put that BG prop -- https://github.com/shapeshift/web/blob/380c1696dfc77132cdbc68ace55249542217881c/src/features/defi/providers/foxy/components/FoxyManager/Deposit/FoxyDeposit.tsx#L524